StockroomRobot.md

```diff
@@ -2,10 +2,10 @@
 # Name: Stockroom Robot
 
 ## Task 1: Get Item From Back
-an item retrieved is a happiness concieved  
+an item retrieved is happiness concieved  
 
 ## Task 2: Get Coffee From Back
- 
+getting coffee from the back makes customers come back 
```

Comment:
rhyming back with back isn't very poetic don't you think?

Reply:
What a conundrum, thank you for that